app/components/IdentityCard/FailedIdentityDetails.tsx

```diff
@@ -83,58 +89,50 @@ ${getOS()}`;
         );
     }
 
+    if (mail === undefined) {
+        return (
+            <div className={clsx('body3 p20', styles.failedDetails)}>
+                <SimpleErrorModal
+                    show={failedConnect}
+                    header="Unable to load identity provider email. Please check your internet connection."
+                    onClick={() => setFailedConnect(false)}
+                />
+                {failedDetails}
+                <Loading className="pV20" inline />
```

Comment:
If the loading fails, i.e. due to a missing internet connection, then the "Loading" is shown forever, even though we never do any retry to get the mail.